app/code/Magento/Sales/Block/Adminhtml/Order/View.php

```diff
@@ -230,9 +231,9 @@ protected function _construct()
             );
         }
 
-        if ($this->_isAllowedAction(
-            'Magento_Sales::ship'
-        ) && $order->canShip() && !$order->getForcedShipmentWithInvoice()
+        if (!$order->getForcedShipmentWithInvoice()
```

Comment:
We still have the order of the operations changed. The conditions order should be the same as we have in PR to 2.2-develop branch. 
Thanks

Reply:
Done